backend/events/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+# Create your models here.
+
+class Event(models.Model):
+    title = models.CharField(max_length=80)
+    description = models.TextField(blank=True, null="True")
+    start_time = models.DateTimeField(null="False")
+    end_time = models.DateTimeField(null="False")
+    event_image = models.ImageField(upload_to=None, default = 'https://via.placeholder.com/150', null="True")
```

Comment:
We're going to need to have this uploading somewhere before I'm comfortable merging it. I believe @scottgall started working on this yesterday; please talk to him today or this weekend and get remote file storage added here.

Reply:
@ShawnArd